levels/pages/faq/faq.jade

```diff
@@ -335,3 +337,48 @@ block content
                 <div class='block__elem3'></div>
             </div>
             ```
+
+        a(id="global-css-resets")
+        .raw-text
+          :markdown
+            ## I've heard that BEM does not recommend global CSS resets. Why?
+
+            > CSS resets is a practise making a good showing. Many frameworks first align anything and then
+            > apply their special styles. BEM does not recommend common resets. Why? And what we are supposed
+            > to do instead?
+
+            Nothing bad would happen to your blocks if you use common reset (well, except of some special cases below).
+            So, BEM does not prohibit to use them. But using them BEM-way would be more effective.
+
+            Common CSS reset is a set of CSS to be applied to document nodes and ensure that their default view is the
+            same in different browsers. In most cases the CSS rules are written for tag selectors and this is not recommended
+            in BEM (you can find a lot of explanation above).
+
+            Another point is that in BEM a block encapsulates everything which is needed for it to be displayed and function.
+            And this is why we call the BEM blocks independent. If the block does not look properly without a third-party
+            CSS being added onto the page, it cannot be called "independent§" that much.
+
+            Assuming this all, BEM recommends every block to reset itself. If you have `menu` block and `list` block both
```

Comment:
This approach will only work if you use a CSS optimizer with structural optimizations. Otherwise you will get too many duplicated code in your output CSS. We cannot recommend CSSO as a structural optimizer because it is frozen. So we propose another way — preprocessor mixins. Besides mixing the reset code it also guaranties that the output will be without duplicant but:
```
.menu, .list {
    padding: 0;
    margin: 0;
}
.menu {
    background: red;
}
.list {
    font-size: 12px;
}
```
Maybe this should be explained in the answer?